client/src/App.css

```diff
@@ -1,38 +1,36 @@
 .App {
-  text-align: center;
+  margin: auto;
+  width: 35%;
```

Comment:
Why the percentages? They don't scale so well to either large or small screens

Reply:
So, what kind of measure you expect? vw?